rnd/autogpt_server/test/executor/test_manager.py

```diff
@@ -15,13 +15,14 @@ async def execute_graph(
     test_user: User,
     input_data: dict,
     num_execs: int = 4,
+    timeout: int = 60,
 ) -> str:
     # --- Test adding new executions --- #
     response = await agent_server.execute_graph(test_graph.id, input_data, test_user.id)
     graph_exec_id = response["id"]
 
     # Execution queue should be empty
-    result = await wait_execution(test_user.id, test_graph.id, graph_exec_id)
+    result = await wait_execution(test_user.id, test_graph.id, graph_exec_id, timeout)
```

Comment:
@ntindle I still don't understand this, if all other PRs are passing without even need to change this. Why this PR?

Reply:
Is there a code change you introduce in this PR that possibly slows down the graph execution ?

Reply:
I don’t know and at this point I’m not planning on spending ANY more time on this PR. I opened it during the offsite. I do know this value is slowly rising as I’ve had to keep updating this number when master changes.

Reply:
We should never slowly update the number unless we know its reason.
If you don't want to spend any more time on this, you can remove the `code.py`, and revert the test_manager.py to merge this. And park the code.py changes, when you have more time to investigate.

Reply:
It is done, I agree we shouldn't but I figure the block just took a while to execute. There's probably better ways and interfaces for code and this was a mostly fun PR for outside work idea stuff that didn't really pan out